scripts/update_tags_per_languages.py

```diff
@@ -282,54 +312,141 @@ def update_tags_field(tags_field_string: str, tags_field_lc: str, current_tag: s
+            sys.exit()
+
+    if env == "prod":
+        dev = False
+        env = "org"
+        user = ""
+    elif env == "dev":
+        dev = True
+        env = "net"
+        user = "off:off@"
+    else:
+        print("Environment should be 'prod' or 'dev', unexpected value:", env)
+        sys.exit()
+
     for plural, singular in map_tags_field_url_parameter.items():
         # 0) set variables
         tags_list_url = f"https://{user}world.openfoodfacts.{env}/{plural}.json"
+        # reinitialize for each loop because "plural" is added below, hence, we need to remove it for the next item
+        data = {
+            'user_id': args.user_id,
+            'password': args.password,
+        }
         
         # by default the query return 24 results. 
         # increase to 1000 (so far chips in EN (should be crisps in EN)
-        #   had max number of products for categories with ~550)
+        #   add max number of products for categories with ~550)
         products_list_for_tag_url = f"https://{user}world.openfoodfacts.{env}/{singular}/{{tag_id_placeholder}}.json?page_size=1000"
         
-        file = os.path.abspath(os.path.join(os.path.dirname( __file__ ), '..', f'taxonomies/{plural}.txt')) 
+        taxonomy_file_location = os.path.abspath(os.path.join(os.path.dirname( __file__ ), '..', f'taxonomies/{plural}.txt')) 
         
         # country is needed otherwise <plural>_lc will be "en"
         post_call_url = f"https://{user}{{country}}.openfoodfacts.{env}/cgi/product_jqm2.pl"
 
+        # if log_file_name_2 exists, it means step 1) and 2) completely ran already, hence, resume from step 3)
+        if not os.path.exists(log_file_name_2.format(plural=plural)):
+            # 1) get all tags
+            api_result = get_from_api(tags_list_url)
+            # example:
+            #  api_result = {
+            #      "tags": [
+            #          {"id": "it:frankreich", "known": 0, "name": "frankreich", ...}, 
+            #      ],
+            #  }
 
-        # 1) get all tags
-        all_tags = get_from_api(tags_list_url)
-        # example:
-        #  all_tags = {
-        #      "tags": [
-        #          {"id": "it:frankreich", "known": 0, "name": "frankreich", ...}, 
-        #      ],
-        #  }
+            # 2) fetch unknown tags and look into taxonomy
+            unknown_tags_taxonomy_comparison(api_result, taxonomy_file_location, plural, dev)
+
+            # create second log file
+            with open(log_file_name_2.format(plural=plural), "w"):
+                pass
+
+        # retrieve result of the previous step
+        possible_wrong_language_tags = {}
+        with open(log_file_name_1.format(plural=plural), 'r') as log_file_1:
+            # skip header
+            log_file_1.readline()
+            for line in log_file_1:
+                key, value = line.strip().split(',')
+                possible_wrong_language_tags[key] = value
```

Comment:
nit: fashioned pythonista ;-)

```suggestion
            possible_wrong_language_tags = dict(
                line.strip().split(',', 1) for line in log_file_1
            )
```